src/main/java/picard/sam/SamErrorMetric/CollectSamErrorMetrics.java

```diff
@@ -162,6 +168,15 @@
     @Argument(shortName = "P", doc = "The probability of selecting a locus for analysis (for downsampling).", optional = true)
     public double PROBABILITY = 1;
 
+    @Argument(
+            fullName = "progressStepInterval",
+            doc = "The interval between which progress will be displayed.",
+            optional = true
+    )
+    public int progressStepInterval = 100000;
```

Comment:
the standard for Picard arguments is UPPER_CASED.

Reply:
Fixed!